lib/colorado_lottery.rb

```diff
@@ -0,0 +1,45 @@
+class ColoradoLottery
+  attr_accessor :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.game_interests.include?(game.name) && (contestant.age >= 18)
+  end
+
+  def can_register?(contestant, game)
+
+    interested_and_18?(contestant, game) &&
+    ((game.national_drawing?) || (contestant.state_of_residence == "CO"))
```

Comment:
nicely done-- a couple small points, but you don't need parentheses around `game.national_drawing?` (it's all one method, so it would be evaluated all at once anyway) and you could have used `@contestant.out_of_state?` instead of duplicating that logic here that you already wrote in your contestant class.